rest/oidc_api.go

```diff
@@ -27,6 +28,14 @@ const (
 	OIDC_AUTH_REDIRECT_URI  = "redirect_uri"
 	OIDC_AUTH_STATE         = "state"
 
+	// Request parameter to specify the OpenID Connect provider to be used for authentication,
+	// from the list of providers defined in the Sync Gateway configuration.
+	requestParamProvider = "provider"
```

Comment:
This parameter is intended for the callback URL, not the request URL.

```suggestion
	callbackParamProvider = "provider"
```

Reply:
Actually this parameter was named requestParamProvider since it its being used not only in callback URL but also in /{db}/_oidc request, called by clients to initiate the OIDC Authorization Code flow. I think renaming this to oidcAuthRequestURI makes more sense.